lib/get_weather/weather.rb

```diff
@@ -18,10 +19,14 @@ def get_weather_data_hash
       }
     end
 
+    def epoch_to_local(time)
+      @timezone.to_local(Time.at(time))
```

Comment:
You have an `attr_reader` defined but are using the `@timezone` instance variable directly.  Either remove the `attr_reader` or access it as `timezone` here.

Reply:
Ok sure